README.md

```diff
@@ -1 +1,152 @@
+
+>The Genius Academy is a platform for learning skills, providing educational resources and courses to help individuals improve their knowledge and abilities in a specific area.
+
+**[Genius Academy]** 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+<summary>Client</summary>
+<ul>
+  <li><a href=" HTML Tutorial (w3schools.com) ">HTML</a></li>    
+  <li><a href=" CSS Tutorial (w3schools.com) ">CSS</a></li>
+ </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Github Flow execution]**
+ 
+- **[Implement both Mobile and desktop version]**
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://saluumaa.github.io/GeniusAcademy-institute/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- **github**
+- **terminal**
+-**Code Editor**
+ 
+### Setup
+
+Clone this repository to your desired folder: 
+Use Terminal:
+
+```sh 
+cd my-folder
+ 
+git clone git@github.com:saluumaa/skill-sail-site.git
+```
+### Install
+
+Install this project with:
+```sh
+cd my-project
+ 
+npm install
+```
+
+### Run tests
+
+To run tests, run the following command:
+```sh 
+Npx hint .   
+ 
+npx stylelint "**/*.{css,scss}"     to check errors for CSS file.
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Saluuma Hassan**
+
+- GitHub: [@Saluumaa](https://github.com/saluumaa)
+- Twitter: [@SalmaHIbrahim20](https://twitter.com/SalmaHIbrahim20)
+- LinkedIn: [Salma ibrahim](https://www.linkedin.com/in/salma-ibrahim-78bb5a14a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Web Accessibility]**
+ 
+- [ ] **[Implement Forms Validation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please give a ⭐️ if you like this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank everyone contributed the completion of this project, especially thanks to (<a href="https://www.behance.net/adagio07">Cindy Shin</a>) who is the owner of ther original design<a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015"> 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] as mentioned by both previous reviewers, kindly fix the MIT LICENSE link , it's still broken 🙏 

![image](https://user-images.githubusercontent.com/89282221/217807789-2329f843-1e01-4f7f-86df-105f03557723.png)

Reply:
@ichala, yes but I created the MIT license and merged it to the main branch, so it will not appear in this branch as soon as I merge it, the main branch has the License and it will work there.
kindly recommend to me how to add a readme file in the branch if it is already in the main branch

Reply:
@saluumaa kindly do not merge or update the main branch until you get approved, you can push the changes on this pull request 🙏